chainerrl/wrappers/atari_wrappers.py

```diff
@@ -217,13 +217,28 @@ def _get_ob(self):
 
 
 class ScaledFloatFrame(gym.ObservationWrapper):
+    """Divide frame values by 255.0 and return them as np.float32.
+
+    Especially, when the original env.observation_space is np.uint8,
+    this wrapper converts frame values into [0.0, 1.0] of dtype np.float32.
+    """
+
     def __init__(self, env):
+        assert isinstance(env.observation_space, spaces.Box)
         gym.ObservationWrapper.__init__(self, env)
 
+        self.scale = 255.0
+
+        orig_obs_space = env.observation_space
+        self.observation_space = spaces.Box(
+            low=self._observation(orig_obs_space.low),
+            high=self._observation(orig_obs_space.high),
```

Comment:
I think `low` and `high` should also be scaled, if not -inf or inf.

Reply:
Actually it seems we don't need to consider whether it's inf or not since inf/255 is inf.

Reply:
Yes, `low` and `high` is scaled via `self._observation`.

Reply:
Sorry I was confused. Nevermind.